### PR TITLE
adding schema service entry to the appgw-ingress file

### PR DESCRIPTION
**Pull request by amaverma**
## All Submissions:
-------------------------------------
* [YES/NO] Have you added an explanation of what your changes do and why you'd like us to include them? YES
* [YES/NO] I have updated the documentation accordingly. NA
* [YES/NO/NA] I have added tests to cover my changes. NA
* [YES/NO/NA] All new and existing tests passed. NA
* [YES/NO/NA] I have formatted the terrafrom code.  _(`terraform fmt -recursive && go fmt ./...`)_

## Current Behavior or Linked Issues
-------------------------------------

Adding the entry for Schema service in `azure-common.yaml` file
 
## Does this introduce a breaking change?
-------------------------------------
- [YES/NO]

NO

## Other information
-------------------------------------

cc: @polavishnu4444 , @danielscholl

### CODE REVIEW

File: devops/charts/osdu-common/templates/appgw-ingress.yaml [hunk 1]
```diff
@@ -40,3 +40,7 @@ spec:
               serviceName: search-service
               servicePort: 80
             path: /api/search/v2/*
+          - backend:
+              serviceName: {{ .Release.Name }}-schema
```
Comment (danielscholl) [inline devops/charts/osdu-common/templates/appgw-ingress.yaml:44]: Is there a reason you didn't follow the same pattern as already being used as the "Service Name?"

Please see the pattern in use and continue to use the same pattern.

**Review by danielscholl — CHANGES_REQUESTED** [review]

**Comment by danielscholl** [comment]
@polavishnu4444 

Can we please track in #134 the changes that are coming in.  I need a tracking Issue instead of just blind PR's being submitted.

File: devops/charts/osdu-common/templates/appgw-ingress.yaml (hunk 1 above)
Comment (polavishnu4444) [reply devops/charts/osdu-common/templates/appgw-ingress.yaml:44]: @amaverma - Please do the needful here.

Comment (amaverma) [reply devops/charts/osdu-common/templates/appgw-ingress.yaml]: fixed in the latest iteration

**Review by danielscholl — APPROVED** [review]

**Pull request merged**